src/lib/y2users/user_validator.rb

```diff
@@ -47,21 +48,21 @@ def issues(skip: [])
       list = Y2Issues::List.new
 
       if !skip.include?(:name)
-        err = Yast::UsersSimple.CheckUsernameLength(user.name)
+        err = check_length
         add_fatal_issue(list, err, NAME_LOC)
 
-        err = Yast::UsersSimple.CheckUsernameContents(user.name, "")
+        err = check_characters
         add_fatal_issue(list, err, NAME_LOC)
 
         # Yast::UsersSimple.CheckUsernameConflicts is currently used only when manually creating
         # the initial user during installation, it simply checks against a hard-coded list of
         # system user names that are expected to exist in a system right after installation.
-        err = Yast::UsersSimple.CheckUsernameConflicts(user.name)
+        err = check_username_conflict
```

Comment:
As pointed in the comment, "Yast::UsersSimple.CheckUsernameConflicts is currently used only when manually creating the initial user". So we can remove the Perl function now.

Reply:
thanks for notice, done in https://github.com/yast/yast-users/pull/308